ADVANCED.md

```diff
@@ -1,11 +1,17 @@
-# Use terraform variable files
+# Advanced usage
 
-To avoid using multiple main.tf having the same deployment and just some variables differences, use the tfvars files with a main template to deploy environment.
-For this example, we will have four type of files : 
- - main.tf template, deployment description using variables for environment specific parameters
+## Use terraform variable files
+
+To avoid using multiple `main.tf` files having the same deployment and just some variables differences, use the `tfvars` files with a main template to deploy environment.
```

Comment:
This sentence looks a bit complicated to me. If there are differences in variables, are you sure that deployments are the same in every possible aspect?